lib/galaxy/datatypes/proteomics.py

```diff
@@ -998,49 +998,3 @@ def generate_primary_file(self, dataset=None):
                 rval.append(f'<li><a href="{fn}" type="text/plain">{fn}</a>{opt_text}</li>')
         rval.append('</ul></div></html>')
         return "\n".join(rval)
-
-
-class Analyze75(Binary):
```

Comment:
In theory TS datatypes may have imported from this location ... but I think I'm fine with breaking that.

Reply:
True ... but ... oh :) I hope no one is ever doing it. I moved Analayze75 here as nii1 is supposed to be compatible with it...